app/models.py

```
@@ -1363,6 +1392,25 @@ def distance(self):
+    @property
+    def fuel_consumption(self) -> float | None:
+        """Calculate trip fuel consumption"""
+        if self.end_fuel_level is None or self.start_fuel_level is None:
+            return None
+        return self.start_fuel_level - self.end_fuel_level
+
+    @property
+    def fuel_consumption_human_readable(self) -> str | None:
+        """Calculate trip fuel consumption with a plus sign instead of negative for negative fuel consumption"""
+        consumption = self.fuel_consumption
+        if consumption is None: return None
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)